### DISCUSSION TITLE
Possible to use reshape_projoint() for datasets like "exampleData1_labelled_tibble"?

### DISCUSSION THREAD

**Discussion by fabianhaggerty** [yhoriuchi/projoint#40] [Q&A]
Hello, thanks for this great package.
Is it possible to use `reshape_projoint()` (and in the next step `projoint()`) with datasets that are structured like the example data set `exampleData1_labelled_tibble`? 
The code examples provided only show cases where `exampleData1` is used, which has a different structure.
If it is possible to use `exampleData1_labelled_tibble` for `reshape_projoint()` that would be a great help - especially if you could provide a short code example.
Many thanks!

**Comment by yhoriuchi** [comment]
See my response in #41 .
